front/public/locales/en/operational-studies.json

```diff
@@ -1,4 +1,13 @@
 {
+  "boards": {
+    "trains": "Trains",
+    "sdd": "SDD",
+    "std": "STD",
+    "conflicts": "Conflicts",
+    "map": "Map",
+    "macro": "Macro",
+    "table": "Table"
```

Comment:
This should be "Tables" (plural)

Or is this part of the modifications agreed on with Thibaut? (In which case, there's a discrepancy between French and English: French is plural while English is singular.)

Reply:
Sure tables seem more logical